src/screens/HomeStackScreen.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { createNativeStackNavigator } from "@react-navigation/native-stack";
```

Comment:
This should be moved to navigation, and shouldn't be appended with Screen, lemme know if that makes sense